src/main/java/be/syntra/devshop/DevshopBack/services/SearchService.java

```diff
@@ -0,0 +1,13 @@
+package be.syntra.devshop.DevshopBack.services;
+
+import be.syntra.devshop.DevshopBack.entities.Product;
+import be.syntra.devshop.DevshopBack.models.SearchModel;
+
+import java.util.List;
+
+public interface SearchService {
+
+    void setSearchModel(SearchModel searchModel);
```

Comment:
This method implies state can be stored inside of this service, which probably isn't a good idea since the entire backend should be stateless

Reply:
see answer below

Reply:
fixed